pom.xml

```diff
@@ -87,7 +87,7 @@
 		<mockk.version>1.11.0</mockk.version>
 		<powermock.version>2.0.9</powermock.version>
 		<scala.version>2.12.13</scala.version>
-		<slf4j.version>1.7.30</slf4j.version>
+		<slf4j.version>1.8.0-beta4</slf4j.version>
```

Comment:
I think a stable version of SLF4J 1.8.0 will be never released in favor of 2.0. Could you update the version to the latest 2.0 (alpha) version?

Reply:
Sure thing 👍